change/@ni-nimble-components-9a663b6a-ecb5-4698-bd2c-77cb3281255a.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
The PR description can include a link to the spec but should also be self contained.
Please update the PR description to describe what is implemented in this specific PR. (i.e. added the diesTable, added the WorkerRenderer)